staker-rewards/inspector/scripts/lib/contract-helper.js

```diff
@@ -0,0 +1,87 @@
+import { EthereumHelpers } from "@keep-network/tbtc.js"
+
+const GET_PAST_EVENTS_BLOCK_INTERVAL = 50000
```

Comment:
This is not a limit of the event lookback period. It is used only in case we fail to get the events in one request and we switch to the partial events pulls mode. Then we use `GET_PAST_EVENTS_BLOCK_INTERVAL` to determine the blockspan of each batch.

Reply:
Correct. And my question is, why `50000` is a good value for this constant. I am not aware of any limitations of this kind other than ones imposed by Alchemy. And if we are doing it so that the script can handle Alchemy Ethereum hosts as well, number `50000` is probably wrong. From the docs I linked in the first comment:

> NOTE: You can make eth_getLogs requests with up to a 2K block range and no limit on the response size. You can also request any block range with a cap of 10K logs in the response.

Reply:
`50k` was an arbitrary value chosen to get fewer batches but with a bigger size. I think we can set it to `2k` as well to satisfy Alchemy.